inst/ms/aggreCAT.qmd

```diff
@@ -1293,16 +1378,16 @@ plot_judgements <-
 p <- plot_judgements %>%  
   dplyr::bind_rows(., {dplyr::semi_join(plot_cs, plot_judgements,
                           by = "paper_id")}) %>% 
-  ggplot(aes(x = x_vals, y = y_vals)) +
-  geom_pointrange(aes(xmin = three_point_lower, 
+  ggplot2::ggplot(ggplot2::aes(x = x_vals, y = y_vals)) +
+  ggplot2::geom_pointrange(ggplot2::aes(xmin = three_point_lower, 
                       xmax = three_point_upper)) +
-  facet_grid(data_type ~ paper_id, scales = "free_y") + 
-  theme_classic() +
-  theme(legend.position = "none") +
-  geom_vline(aes(xintercept = 0.5, colour = as.logical(outcome))) +
-  xlab("Probability of Replication") +
-  ylab(element_blank()) +
-  scale_colour_brewer(palette = "Set1")
+  ggplot2::facet_grid(data_type ~ paper_id, scales = "free_y") + 
+  ggplot2::theme_classic() +
+  ggplot2::theme(legend.position = "none") +
+  ggplot2::geom_vline(aes(xintercept = 0.5, colour = as.logical(outcome))) +
+  ggplot2::xlab("Probability of Replication") +
+  ggplot2::ylab(ggplot2::element_blank()) +
+  ggplot2::scale_colour_brewer(palette = "Set1")
 ```
 
 ```{r}
```

Comment:
I like this figure, maybe consider adding to the package as a function? I can see this being useful in elicitation settings (showing round one judgements + aggregates for people not using the platform as an example). Suggestions:
- Reverse factor levels on y axis
- Spacing on x axis tick marks is wrong
- Again I'd go back to `theme_bw()` minimal aesthetic choices + minor tweaks

Reply:
Agree with both points. New issue created #35

Reply:
If we move code to package, any suggestions for evidence to support claim in the preceding paragraph?:

> Finally, creating bespoke user-defined plots is relatively easy --
because [aggreCAT]{.pkg} functions return tidy dataframes, we can easily
manipulate the raw judgements, aggregated Confidence Scores and outcome
data to plot them with [ggplot2]{.pkg} [@ggplot2016] or other
visualisation package. 

Perhaps just state that we have included in package but theoretically could make any plot you wanted?

Reply:
Could split figure in package - one for ratings, one for confidence scores (if have too many papers, then have mess of figure) - provide option for one, the other, both. Or feed it one paper at a time (round 2 view on the platform).

Reply:
But as is, in the paper is fine.